stdlib/public/core/CompilerProtocols.swift

```diff
@@ -21,7 +21,6 @@
 /// Only three types provided by Swift, `Bool`, `DarwinBoolean`, and `ObjCBool`,
 /// conform to `BooleanType`. Expanding this set to include types that
 /// represent more than simple boolean values is discouraged.
-@swift3_migration(renamed="Boolean")
```

Comment:
Okay, I'll leave it as is then.  Lemme know if you think there's anything else I could improve :-)